docker-compose.yml

```diff
@@ -2,22 +2,19 @@ version: "3.6"
 
 services:
   web:
-    container_name: volume
+    container_name: "volume"
```

Comment:
i don't recommend having container_name -- theres always weird errors once you deploy